app/src/main/java/com/github/se/studybuddies/ui/video_call/CallLobbyScreen.kt

```diff
@@ -41,8 +46,24 @@ fun CallLobbyScreen(
 ) {
   LockScreenOrientation(orientation = Configuration.ORIENTATION_PORTRAIT)
```

Comment:
I think you solved the source of the bug and we didn't need anymore the patch to prevent the rotation of the screen !